src/disk_utils.cpp

```diff
@@ -1195,6 +1196,10 @@ int build_disk_index(const char *dataFilePath, const char *indexFilePath, const
     std::string disk_pq_pivots_path = index_prefix_path + "_disk.index_pq_pivots.bin";
     // optional, used if disk index must store pq data
     std::string disk_pq_compressed_vectors_path = index_prefix_path + "_disk.index_pq_compressed.bin";
+    std::string prepped_base =
+        index_prefix_path +
+        "_prepped_base.bin"; // temp file for storing pre-processed base file for cosine/ mips metrics
```

Comment:
Maybe we can extract the preprocessing code into a function like we did for DLVS? https://msasg.visualstudio.com/Bing_and_IPG/_git/ObjectStore?path=/private/Pegasus/src/DssmSearch/randnsg/src/aux_utils.cpp

Reply:
Let us do this clean-up in another PR..